src/js/juju-cards-v1.0.0.js

```diff
@@ -1,7 +1,15 @@
-let jujuCards = () => {
+// Allow the user of the cards to customize the install URL to their own
+// controller using the data-demoDomain attribute on the script tag
+// embedding the juju-cards.js script.
+const customDemoDomain = document.currentScript.getAttribute('data-demodomain');
```

Comment:
Note this has to be outside in order for the currentScript attribute to work properly. So the custom domain is passed in.